.circleci/config.yml

```diff
@@ -146,7 +146,7 @@ jobs:
       NX_E2E_CI_CACHE_KEY: e2e-circleci-<< parameters.os >>
       SELECTED_PM: << parameters.pm >>
       NX_E2E_RUN_CYPRESS: 'false'
-      NX_VERBOSE_LOGGING: 'true'
```

Comment:
This should be done in a separate PR I believe, best to just rollback @vsavkin's commit